scripts/archive/build-release-archives.sh

```diff
@@ -41,6 +41,8 @@ pwd
 ls
 cp ./_build/default/src/app/archive/archive.exe "${BUILD_DIR}/usr/local/bin/coda-archive"
 cp ./_build/default/src/app/archive_blocks/archive_blocks.exe "${BUILD_DIR}/usr/local/bin/mina-archive-blocks"
+cp ./_build/default/src/app/missing_subchain/missing_subchain.exe "${BUILD_DIR}/usr/local/bin/mina-missing_subchain"
+cp ./_build/default/src/app/missing_blocks_auditor/missing_blocks_auditor.exe "${BUILD_DIR}/usr/local/bin/mina-missing_blocks_auditor"
```

Comment:
Rename `mina-missing_blocks_auditor` and  `mina-missing_subchain` to have all `-` instead of `_`? Seems a bit odd with the way it is now